src/main/kotlin/no/nav/familie/ef/sak/dummy/TestTilkjentYtelseController.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.familie.ef.sak.dummy
```

Comment:
Hvis dere ikke trenger endepunktene i preprod, så hadde jeg foretrukket at dummy-katalogen lå under src/test i stedet for src/main

Reply:
Vi trenger de i preprod for å teste avstemming